bin/diagnostic.js

```diff
@@ -1 +1,10 @@
 'use strict'
+
+const ask = require('../lib/ask.js')
+
+let number = ask("Guess a number!")
+if (number >= 1 && number <= 10) {
```

Comment:
Looks good for the most part, but you want to give the user a specific number to guess in addition to checking to see if their answer is between 1-10